src/Common/src/CoreLib/System/Span.NonGeneric.cs

```diff
@@ -549,4 +549,4 @@ internal static unsafe void ClearWithReferences(ref IntPtr ip, nuint pointerSize
             }
         }
     }
-}
+}
```

Comment:
Nit: file not ending with new line

Reply:
Will this get fixed automatically when we merge this change with the mirror PR from coreclr?

Reply:
Along with the spacing/formatting fixes in src/Common/src/CoreLib/System/Span.cs in here.
I tried to revert these changes by copying the raw files from GitHub to Visual Studio, but it auto formats it. I could use VS Code or a plain text editor and clean up these changes if you think it would be easier to merge.